checkers/Board.py

```diff
@@ -98,14 +115,35 @@ def move_piece(self, mouse_x, mouse_y):
             self.grid[self.selected_piece[0]][self.selected_piece[1]].selected = False
             self.grid[selected_row][selected_col] = self.grid[self.selected_piece[0]][self.selected_piece[1]]
             self.grid[self.selected_piece[0]][self.selected_piece[1]] = 0
+
+
```

Comment:
There is a bug in the code, if a user is selected the piece and change his mind, and selects another piece, an unnecessary piece is eaten. I assume it's because of not clearing _self.opponent_piece_eaten_ attribute if the user changes his mind? Please double check it. Thanks.